.dev/tests/jest/setup-globals.js

```diff
@@ -10,7 +10,7 @@ global.wp = {};
 // Prevent console messages when running tests.
 console = {
 	...console,
-	log: () => { },
+	// log: () => { },
```

Comment:
This was enabled to prevent an abundance of messages specifically around successful block deprecations when running our unit tests. It would be better to determine if we still need this or not and remove it entirely if we don't.

Reply:
Good call JR. Pushed up some changes here. Most of the superfluous console messages are typed `info` and `warn` where we see block deprecation success messages as well as deprecated component usage notices. I suggest that we allow logging for dev debug logs when writing tests.